specs/0113-light-theme.md

```diff
@@ -0,0 +1,35 @@
+- Feature Name: Light theme
+- Created: 2023-06-22
+- Associated Issue: #2141
+
+# Summary
+
+Implement light theme and enable users to switch between the (current) dark theme and the light theme.
+
+# Motivation
+
+Over the years, many users have expressed preference for a light color theme over the Pontoon default dark theme. Having the ability to switch themes will also allow us to implement high contrast themes, which benefit low vision users, increase readability and reduce website’s visual noise.
+
+# Implementation notes
+
+In the first step, we'll replace existing hardcoded color values with CSS variables. We'll store those in separate CSS files, with one version of each file for each theme. Users will be able to switch theme in their settings.
+
+See below for the actual light theme color values.
```

Comment:
This looks very limited given how large Pontoon is. Should these specs be done after we `replace existing hardcoded color values with CSS variables`, and have an actual measure of how many colors we need to replace (and maybe change for consistency)?

Reply:
I've changed the format of the color values, but the list actually covers the translate view and dashboards pretty well. The expectation is that we'll be able to keep colors like status icons the same across both themes.

I wouldn't block the spec on factoring out the color values, because that won't take us to the finish line either. It's totally possible we'll discover that we need a broader spectrum of colors as part of developing the light theme.

These spec is a bit special, because unlike in the feature specs, the idea here is to have a foundation on which to work on the designs for the whole app. Creating a spec with all the designs would be great, but that would essentially solve the vast majority of the implementation.